src/sgame/sg_client.cpp

```diff
@@ -1126,8 +1126,7 @@ const char *ClientConnect( int clientNum, bool firstTime )
 	// check for a password
 	value = Info_ValueForKey( userinfo, "password" );
 
-	if ( g_password.string[ 0 ] && Q_stricmp( g_password.string, "none" ) &&
-	     strcmp( g_password.string, value ) != 0 )
+	if ( g_needpass.Get() && !Str::IsIEqual( g_password.Get(), value ) )
```

Comment:
Do we really want to check passwords in a case-insensitive way?

```suggestion
	if ( g_needpass.Get() && g_password.Get() != value )
```

Reply:
Nope!